lib/fastlane/plugin/ionic/actions/ionic_action.rb

```diff
@@ -253,6 +263,28 @@ def self.available_options
             description: "Specifies whether to run `ionic cordova prepare` before building",
             default_value: true,
             is_string: false
+          ),
+          FastlaneCore::ConfigItem.new(
```

Comment:
those first two are unrelated options I didn't merge over yet, correct?

Reply:
indeed. It's old stuff. But it's not bad to add it now :)

Reply:
yes, the cherry pick bring it too. I saw them and as @blackholegalaxy say it's not bad to add it now. If I remember well (let me the benefit of the doubt) that cordova flag is usefull when you need to restore plugins and platform from scratch and you have one or more plugins from a git source. With latest cordova version it fail and that flag can workaround the error.

Reply:
Isn't this missing the rest of the code for that functionality then?

Reply:
owww. Didn't see the rest of code was missing. Indeed we have to plug the logic for fetch. The min SDK version has no proper logic in the original plugin.

```
def self.check_platform(params)
        platform = params[:platform]
        if platform && !File.directory?("./platforms/#{platform}")
          if params[:cordova_no_fetch]
            sh "cordova platform add #{platform} --nofetch"
          else
            sh "cordova platform add #{platform}"
          end
        end
      end
```

Reply:
@skaman

Reply:
I think i need to cherry pick all this commit: https://github.com/bamlab/fastlane-plugin-cordova/commit/35ab44c529e785f0fc595d8f9349c5f2da04ceee

Reply:
And maybe also this one: https://github.com/bamlab/fastlane-plugin-cordova/commit/605e9b72914e33ae2b527c279442f457ebf1678b

Reply:
👍

Reply:
@janpio @blackholegalaxy I picked that commits and i tested it. It look work correctly with my projects.

Also, about the nofetch feature, the issue described yesterday is better explained here https://github.com/ionic-team/ionic/issues/13205 and here https://issues.apache.org/jira/browse/CB-13507

I've a project with that problem (to workaround the issue on my CI/CD i use an older version of cordova). So I tested this nofetch feature on that project and it look (finally) restore my plugins correctly.

Reply:
nope, i missed an error on my log. It doesn't work. Please wait to pull. It's broken now :(

Reply:
ok no, my bad sorry, it work. I think I had something broken in my gemfile. After a "bundle update" it started to compile fine. So I guess my issue was not related to these commits.